jobs/gorouter/templates/gorouter.yml.erb

```diff
@@ -387,4 +388,4 @@ if_p('router.html_error_template') do |t|
 end
 
 params.to_yaml[3..-1]
-%>
+%>
```

Comment:
please re-add the newline at the end of the file

Reply:
Done.